chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,782 @@
+so you first have to turn many variables into a single variable
+by using an index or a propensity score.\sidenote{
+	\url{https://dimewiki.worldbank.org/Propensity\_Score\_Matching}}
+The \texttt{iematch}\sidenote{
+	\url{https://dimewiki.worldbank.org/iematch}}
+command in the \texttt{ietoolkit} package developed by DIME Analytics
+produces matchings based on a single continuous matching variable.
+
+%-----------------------------------------------------------------------------------------------
+\subsection{Structuring datasets over multiple time periods}
+
+Your data map and master dataset requirements also depends on
+whether you are using data from one time period or several.
+A study that observes data in only one time period is called
+a \textbf{cross-sectional study}.
+\index{cross-sectional data}
+This type of data is relatively easy to collect and handle because
+you do not need to track individuals across time.
+Instead, the challenge in a cross-sectional study is to
+show that the control group is indeed a valid counterfactual to the treatment group.
+
+A study that observes data in multiple time periods is called \textbf{longitudinal}\index{longitudinal data}
+and can either be a \textbf{repeated cross-sections}
+or a \textbf{panel data} study.\index{panel data} \index{repeated cross-sectional data}
+In repeated cross-sections,
+each successive round of data collection uses a new random sample
+of observations from the treatment and control groups,
+but in a panel data study the same observations are tracked and included each round.
+If you are using panel data study,
+then your data map must document how the different rounds will be merged or appended,
+and your master dataset will be the authoritative source of which ID that will be used. 
+
+You must keep track of the \textit{attrition rate} in longitudinal data,
+which is the share of observations not observed in follow-up data.
+It is common that the observations not possible to track
+can be correlated with the outcome you study.
+For example, poorer households may live in more informal dwellings,
+patients with worse health conditions might not survive to follow-up,
+and so on.
+If this is the case, then your results might only be an effect of your remaining sample
+being a subset of the original sample that were better or worse off from the beginning.
+You should have a variable in your master dataset that indicates attrition.
+A balance check using the attrition variable can provide insights
+as to whether the lost observations were systematically different
+compared to the rest of the sample,
+and there are a variety of methods for estimating treatment effects
+with selective attrition.
+
+%-----------------------------------------------------------------------------------------------
+\subsection{Monitoring data}
+
+Another type of data that is, in certain research designs,
+very important for the validity of your research is \textbf{monitoring data}.\sidenote{
```

Comment:
suggest changing to: 'For any study with an ex-ante design, monitoring data is very important for understanding whether field realities match the research design'

Reply:
I like it - implemented in 1e838c7fd502049558fa5b91985cd673d542ed75